napoli-pain/src/components/order_details/order_details.rs

```diff
@@ -90,6 +117,31 @@ impl Component for OrderDetails {
                 self.order = Some(order);
                 true
             }
+            Self::Message::StreamingConnected(stream) => {
+                self.live_streaming_status = LiveStreamingStatus::Connected;
+                ctx.link()
+                    .send_stream(stream.map(|single_order_reply_result| {
+                        match single_order_reply_result {
+                            Ok(SingleOrderReply { order: Some(o) }) => {
+                                return Self::Message::GotStreamingOrderUpdate(o);
+                            }
+                            Ok(SingleOrderReply { order: None }) => Self::Message::StreamingFailed(
+                                service::ServiceError::from("Got empty order"),
+                            ),
+                            Err(e) => {
+                                println!("Error in stream: {:?}", e);
```

Comment:
I don't think this `println!()` is properly sent to the browser console, or is it?